### PR TITLE
[Reviewer: Alex] Add high-level SAS log on registration expiry timer pop

### PR DESCRIPTION
**Pull request by rkd-msw**
Follow-on from https://github.com/Metaswitch/sprout/pull/1329, just making the detailed timeline view of this SAS log useful at a glance.

### CODE REVIEW

File: clearwater_sas_resource_bundle.yaml [hunk 1]
```diff
@@ -801,6 +801,22 @@ events:
       Unable to deregister with an application server as part of a Registration Termination Request. Served User: '{{ var_data[0] }}'
     level:   80
 
+  0x81008B:
+    summary: 'Registration expired for Public ID "{{ var_data[0] }}"'
+    details: |
+      All contact bindings for {{ var_data[0] }} have expired and will be removed. The subscriber will be deregistered at the HSS.
+
+      This was triggered by a Chronos timer pop.
```
Comment (AlexHockey) [inline clearwater_sas_resource_bundle.yaml]: I'd rather not refer to chronos in a level 80 log (we try not to expose chronos too much in SAS as its not a separate node type).

File: clearwater_sas_resource_bundle.yaml [hunk 2]
```diff
@@ -801,6 +801,22 @@ events:
...
+    level:   80
+
+  0x81008C:
+    summary: 'Some bindings expired for Public ID "{{ var_data[0] }}"'
+    details: |
+      Some (but not all) contact bindings for {{ var_data[0] }} have expired and will be removed.
```
Comment (AlexHockey) [inline clearwater_sas_resource_bundle.yaml]: Worth saying that the subscriber is still registered?

**Comment by AlexHockey** [comment]
Looks good, a couple of minor comments.

File: clearwater_sas_resource_bundle.yaml (hunk 1 above)
Comment (rkd-msw) [inline clearwater_sas_resource_bundle.yaml]: Are you happy with just "by a timer pop", or should I remove this line entirely?

Comment (AlexHockey) [inline clearwater_sas_resource_bundle.yaml]: I'm happy with "by a timer pop" but I'm not sure it's necessary. The fact they have "expired" probably tells the reader enough.

**Pull request merged**